server/seeds/index.js

```diff
@@ -1,11 +1,25 @@
 const db = require("../config/connection");
-const { Song } = require("../models");
+const { Song, User, Rating } = require("../models");
 const songSeeds = require("./songSeeds.json");
+const userSeeds = require("./userSeeds.json");
+const ratingSeeds = require("./ratingSeeds.json");
+const songs = Song.create(songSeeds);
+const users = await User.create(userSeeds);
+const newRating = await Rating.create({
+  rating: 3,
+  user: users[0]._id,
+  song: songs[0]._id,
+});
 
 db.once("open", async () => {
   try {
     await db.dropDatabase();
-    await Song.create(songSeeds);
+    await songs;
+    await users;
+    await newRating;
+    await Song.findByIdAndUpdate(user[0]._id, {
```

Comment:
This should update User, not Song. user[0] doesn't refer to anything, did you mean users[0]?